docs/reference/api.md

```diff
@@ -14,35 +14,35 @@ For more examples, see "How to use Pebble API". <!-- [David] Link to the how-to
 
 To access the API endpoints over the Unix socket, use the `--unix-socket` option of `curl`. For example:
 
-* TODO
-
-    ```
-    curl ...
-    ```
+```{terminal}
+   :input: curl --unix-socket /path/to/.pebble.socket -XPOST http://localhost/v1/services -d '{"action": "stop", "services": ["svc1"]}'
+{"type":"async","status-code":202,"status":"Accepted","change":"42","result":null}
+```
 
-* TODO
+<br />
 
-    ```
-    curl ...
-    ```
+```{terminal}
+   :input: curl --unix-socket /path/to/.pebble.socket -XGET localhost:4000/v1/changes/42/wait
+{"type":"sync","status-code":200,"status":"OK","result":{"id":"42","kind":"stop","summary":"Stop service \"svc1\"","status":"Done","tasks":[{"id":"42","kind":"stop","summary":"Stop service \"svc1\"","status":"Done","progress":{"label":"","done":1,"total":1},"spawn-time":"2025-01-10T10:59:53.903366896+08:00","ready-time":"2025-01-10T10:59:53.930409771+08:00"}],"ready":true,"spawn-time":"2025-01-10T10:59:53.903373188+08:00","ready-time":"2025-01-10T10:59:53.930410979+08:00"}}
```

Comment:
Do you think we need the whole response here? I think it's better to use `...`, like:

```
{"type":"sync","status-code":200,"status":"OK","result":{...}}
```

As the result isn't interesting or important here (when you're waiting for a known change).